src/stages/sqs-lambda.ts

```diff
@@ -58,6 +58,7 @@ export class SqsToLambdaStage extends DataStage {
         environment: {
           EVENT_SOURCE: eventSource,
           EVENT_DETAIL_TYPE: eventDetailType,
+          ...(functionProps.environment ?? {}),
```

Comment:
I also realized we weren't respecting the environment variables that the customer set.